impl/src/main/java/org/ehcache/impl/internal/store/heap/OnHeapStore.java

```diff
@@ -1678,6 +1715,8 @@ public void releaseStore(Store<?, ?> resource) {
           throw new IllegalStateException("Exception while releasing Copier instance.", e);
         }
       }
+      StatisticsManager.nodeFor(onHeapStore).clean();
```

Comment:
because the code above can throw, this line and the following should be before the block releasing `Copier`s